docs/protocol-design/introduction.md

```diff
@@ -49,6 +49,37 @@ Although there is cross-over between the two main areas of the living whitepaper
+
+The original Nano (RaiBlocks) paper and first beta implementation were published in December, 2014, making it one of the first Directed Acyclic Graph (DAG) based cryptocurrencies [^9]. Soon after, other DAG cryptocurrencies began to develop, most notably DagCoin/Byteball and IOTA [^10], [^11]. These DAG-based cryptocurrencies broke the blockchain mold, improving system performance and security. Byteball achieves consensus by relying on a “main-chain” comprised of honest, reputable and user-trusted “witnesses”, while IOTA achieves consensus via the cumulative PoW of stacked transactions. Nano achieves consensus via a balance-weighted vote on conflicting transactions. This consensus system provides quicker, more deterministic transactions while still maintaining a strong, decentralized system. Nano continues this development and has positioned itself as one of the highest performing cryptocurrencies.
+
+## References
+
+[^1]: "Block Confirmation Times", 2020. [Online]. Available: https://repnode.org/network/confirmation
+[^2]: "Nano Stress Tests - Measuring BPS, CPS, & TPS in the real world", 2020. [Online]. Available: https://forum.nano.org/t/nano-stress-tests-measuring-bps-cps-tps-in-the-real-world/436
+[^3]: S. Nakamoto, “Bitcoin: A peer-to-peer electronic cash system,” 2008. [Online]. Available: http://bitcoin.org/bitcoin.pdf
+[^4]: “Bitcoin median transaction fee historical chart.” [Online]. Available: https://bitinfocharts.com/comparison/bitcoin-median-transaction-fee.html
+[^5]: “Bitcoin average confirmation time.” [Online]. Available: https://www.blockchain.com/charts/avg-confirmation-time
+[^6]: "Irreversible Transactions - How many confirmation are required", 2020. [Online]. Available: https://en.bitcoin.it/wiki/Irreversible_Transactions#How_many_confirmations_are_required
+[^7] "Bitcoin Energy Consumption Index", 2020. [Online]. Available: https://digiconomist.net/bitcoin-energy-consumption/
+[^8]: S. King and S. Nadal, “Ppcoin: Peer-to-peer crypto-currency withproof-of-stake,” 2012. [Online]. Available: https://peercoin.net/assets/paper/peercoin-paper.pdf
```

Comment:
I can't get this link to load, is there a good alternative?

Reply:
> Thanks for helping sort this out @qwahzi! I made a few requests for changes, but I think this is a good start to get some content rolling on the page. It may still need something thinking through how best to split up some of the pieces better between Abstract, Introduction and Background sections, as some of it felt doubled-up or misplaced. I plan to take a closer look later but will likely merge this after the suggest changes are made.

No problem, and I definitely agree with your comment here. I was trying to port over as much of core content/structure of the original whitepaper as I could, but I'm still struggling with placement, structure, and minimization of duplicate content. Some potential changes for future consideration:

- Merge the 1st Introduction paragraph with the 1st Abstract paragraph, remove the Introduction paragraphs before the horizontal rule, and then move the 2nd two Abstract paragraphs into Introduction? That would make Abstract one medium-sized paragraph with high level claims and not too many specific details, allowing the introduction to expand on those details a little more, while also removing some duplicate/unimportant information. Example:

> ## Abstract

> Since Bitcoin's release in 2009, there has been a growing shift away from traditional, government-backed currencies and financial systems towards modern payments systems based on cryptography, which offer the ability to store and transfer funds in a trustless and secure manner [^3]. In order to function effectively, a currency must be easily transferable, non-reversible, and have limited or no fees. Unfortunately, increased transaction times, high fees, limited network scalability, and high energy consumption have raised questions about the practicality of Bitcoin as an everyday currency.  Here we introduce Nano, a cryptocurrency with a novel block-lattice architecture where each account has its own blockchain, enabling near instant transactions and scalability that is not artificially limited by protocol-side variables like block sizes or block times. 

> ## Introduction

> Nano is a low-latency, feeless, scalable, and environmentally friendly cryptocurrency that improves on many of Bitcoin's core properties via unique design decisions. For example, each Nano user has their own blockchain, allowing them to update their chain asynchronously vs other transactions on the network, resulting in fast transactions with minimal overhead. Transactions keep track of account balances rather than transaction amounts, allowing aggressive database pruning without compromising security. Consensus is maintained by [Open Representative Voting (ORV)](../glossary.md#open-representative-voting-orv), which facilitates irreversible finality (full-settlement). User-selected representative nodes vote on each transaction, and every node independently [cements](../glossary.md#cementing) each transaction after seeing enough representative votes to achieve [quorum](../glossary.md#quorum).

> To date, the Nano network has processed more than 53 million transactions with an unpruned ledger size of only 25.33GB. Average transaction confirmation time during typical network conditions is 0.2 seconds [^1]. The production network has seen traffic as high as 161 CPS (80.5-161 TPS), while the beta network has achieved >1800 CPS (900-1800 TPS) [^2]. Nano’s feeless, split-second transactions make it an ideal cryptocurrency for consumer transactions, while also maintaining decentralization, censorship-resistance, and self-sovereignty.

- Moving (or even removing) the "The following sections of the Living Whitepaper..." section to a new section with its own header, maybe before even the abstract?

- Removing the ORV/PoS comparison piece and just linking to the ORV page (has the same bullet point comparison list already)?

Reply:
> I can't get this link to load, is there a good alternative?

Replaced with: https://decred.org/research/king2012.pdf